ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -121,12 +121,29 @@ public override void OnCollectExtraUpdates(UIViewOperationQueue uiViewOperationQ
 
             if (_inline != null)
             {
+                var _paddings = new Thickness()
+                {
```

Comment:
The thickness is only used to position the text correctly. The layout takes care of the required dimensions.